src/Scheduler.ts

```diff
@@ -1,11 +1,59 @@
-import {Subscription} from './Subscription';
 import {Action} from './scheduler/Action';
+import {Subscription} from './Subscription';
+
+/**
+ * An execution context and a data structure to order tasks and schedule their
+ * execution. Provides a notion of (potentially virtual) time, through the
+ * `now()` getter method.
+ *
+ * Each unit of work in a Scheduler is called an {@link Action}.
+ *
+ * ```ts
+ * class Scheduler {
+ *   now(): number;
+ *   schedule(work, delay?, state?): Subscription;
+ * }
+ * ```
+ *
+ * @class Scheduler
+ */
```

Comment:
For anyone else reviewing this PR: JSDoc in this file looks correct and should render correctly on the site. 👍